imagery/i.cluster/i.cluster.md

```diff
@@ -89,22 +86,22 @@ by *[g.gui.iclass](g.gui.iclass.md)*). The purpose of seed signatures is
 to optimize the cluster decision boundaries (means) for the number of
 clusters specified.
 
-**sample=***rows,cols*  
+**sample:**
 These numbers are optional with default values based on the size of the
```

Comment:
With this change, what does "these numbers" refer to?
That's what I was trying to say in the second part of 
https://github.com/OSGeo/grass/pull/5120#discussion_r1957226110

Reply:
I forgot to answer this... My idea was to simplify the look of it, the type of parameter is already in the synopsis, so I felt it was duplicating the information.

Reply:
Ah, ok, sure, if it's at the top..